docs/tutorials/google/start.ipynb

```diff
@@ -1,478 +1,532 @@
 {
```

Comment:
<p>s/medated/mediated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>replace </p><p> </p><p>Google's Quantum Research Platform enables researchers to run quantum programs on Google's quantum processors. Access is mediated through the Quantum Engine service. </p><p> </p><p>with</p><p> </p><p>Access to Google's quantum processors is mediated through the Quantum Engine service.</p>